### PR TITLE
Add MAC CI

### PR DESCRIPTION
**Pull request by jeremad**
## Summary
Make the codebase compile with homebrew packages, because travis infrastructure uses this

Closes: #4 #7

### CODE REVIEW

File: cmake/r8b/CMakeLists.txt [hunk 1]
```diff
@@ -48,3 +48,4 @@ if(WINDOWS)
 endif(WINDOWS)
 
 set(R8B_LIBRARY r8b PARENT_SCOPE)
+message(STATUS "fin r8b")
```
Comment (w-m) [inline cmake/r8b/CMakeLists.txt]: Debugging artefact, can be removed now?

**Comment by w-m** [comment]
:apple: :tada:

File: cmake/r8b/CMakeLists.txt (hunk 1 above)
Comment (jeremad) [reply cmake/r8b/CMakeLists.txt]: yep

**Comment by klem23** [comment]
Excellent ! (with Mr Burns voice)

**Pull request merged**